src/python/grpcio/grpc/aio/_channel.py

```diff
@@ -115,13 +116,15 @@ def __init__(
 
     @staticmethod
     def _init_metadata(
-        metadata: Optional[Metadata] = None,
+        metadata: Optional[MetadataType] = None,
         compression: Optional[grpc.Compression] = None,
     ) -> Metadata:
         """Based on the provided values for <metadata> or <compression> initialise the final
         metadata, as it should be used for the current call.
         """
         metadata = metadata or Metadata()
+        if not isinstance(metadata, Metadata):
```

Comment:
Can we be more specific and actually check whether this is a tuple with the right shape?